src/Components/Patient/ManagePatients.tsx

```diff
@@ -734,13 +734,21 @@ export const PatientManager = () => {
             <ButtonV2
               id="add-patient-details"
               onClick={() => {
+                const showAllFacilityUsers = ["DistrictAdmin", "StateAdmin"];
```

Comment:
A state admin has access only to the facility belonging to the users state. 
A district admin has access only to the facility belonging to the users district.